app/script/util/DebugUtil.js

```diff
@@ -23,170 +23,166 @@ window.z = window.z || {};
-      .then(_got_notifications);
+    return wire.app.service.notification.getNotifications(clientId, notificationIdSince, 10000).then(_gotNotifications);
   }
 
-  get_notifications_from_stream(remote_user_id, remote_client_id, matching_notifications = [], notification_id_since) {
-    const local_client_id = wire.app.repository.client.current_client().id;
-    const local_user_id = wire.app.repository.user.self().id;
+  getNotificationsFromStream(remoteUserId, remoteClientId, matchingNotifications = [], notificationIdSince) {
+    const localClientId = wire.app.repository.client.current_client().id;
+    const localUserId = wire.app.repository.user.self().id;
 
-    const _got_notifications = ({has_more, notifications}) => {
-      const additional_notifications = !remote_user_id
+    const _gotNotifications = ({hasMore, notifications}) => {
+      const additionalNotifications = !remoteUserId
         ? notifications
         : notifications.filter(notification => {
             const {payload} = notification;
             for (const {data, from} of payload) {
-              if (data && [local_user_id, remote_user_id].includes(from)) {
+              if (data && [localUserId, remoteUserId].includes(from)) {
                 const {sender, recipient} = data;
-                const incoming_event = sender === remote_client_id && recipient === local_client_id;
-                const outgoing_event = sender === local_client_id && recipient === remote_client_id;
+                const incoming_event = sender === remoteClientId && recipient === localClientId;
+                const outgoing_event = sender === localClientId && recipient === remoteClientId;
                 return incoming_event || outgoing_event;
               }
             }
             return false;
           });
 
-      matching_notifications = matching_notifications.concat(additional_notifications);
+      matchingNotifications = matchingNotifications.concat(additionalNotifications);
 
-      if (has_more) {
-        const last_notification = notifications[notifications.length - 1];
-        return this.get_notifications_from_stream(
-          remote_user_id,
-          remote_client_id,
-          matching_notifications,
-          last_notification.id
+      if (hasMore) {
+        const lastNotification = notifications[notifications.length - 1];
+        return this.getNotificationsFromStream(
+          remoteUserId,
+          remoteClientId,
+          matchingNotifications,
+          lastNotification.id
         );
       }
 
-      if (remote_user_id) {
+      if (remoteUserId) {
         this.logger.log(
-          `Found '${
-            matching_notifications.length
-          }' notifications between '${local_client_id}' and '${remote_client_id}'`,
-          matching_notifications
+          `Found '${matchingNotifications.length}' notifications between '${localClientId}' and '${remoteClientId}'`,
+          matchingNotifications
         );
```

Comment:
Let's move the log messag into a const so we can turn four lines into 2.

Reply:
The message is still too long and then it breaks it in a worst way..
![code](https://user-images.githubusercontent.com/125676/37590305-87419bb6-2b70-11e8-944c-978f64ebdb52.png)

Reply:
Ok, will take a look how to shorten the message to make it fit.